WordPress/gutenberg
# Keep an inserted pattern within view when added to the page

## What?

Fixes https://github.com/WordPress/gutenberg/issues/59962

## Why?

Currently, when we add a pattern via the Inserter, the pattern is added to the bottom of the page and cannot be seen. This PR resolves the issue and makes the pattern visible on insert, allowing the user to see the newly added pattern.
 
## Testing Instructions
- Go do editor
- Insert pattern
- Observe that your pattern in view
 
## Screenshots or screencast 

https://github.com/WordPress/gutenberg/assets/61308756/ed13acb2-c4a5-41c8-aea6-0352e2b86fc2
